core/templates/components/skill-selector/skill-selector.component.html

```diff
@@ -0,0 +1,120 @@
+<div class="skill-selector">
+  <div class="skill-selector-parent skill-container oppia-skill-container protractor-test-skill-container">
+    <div class="list-group oppia-list-group">
+      <mat-card layout="row" class="list-view-item protractor-test-skill-select-header">
+        Search By Skill
+        <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" [(ngModel)]="skillFilterText" autofocus>
+      </mat-card>
+      <div class="skill-list-box">
+        <mat-radio-group [(ngModel)]="selectedSkill" (change)="setSelectedSkillId()">
+          <div *ngFor="let topic of currCategorizedSkills | keyvalue">
+            <!-- topic.key = topicName, topic.value = subtopics -->
+            <mat-card class="list-view-item" *ngIf="checkTopicIsNotEmpty(topic.key)">
+              <mat-card layout="row" class="list-view-item">
+                <h2><label class="oppia-topic-name-text">{{ topic.key }}</label></h2>
+              </mat-card>
+              <div *ngFor="let subtopic of topic.value | keyvalue" class="list-view-item ">
+                <!-- subtopic.key = subtopicName (string) / uncategorized, subtopic.value = skills (ShortSkillSummary[]) -->
+                <div *ngIf="!checkIfEmpty(subtopic.value)">
+                  <mat-card class="list-view-item">
+                    <label class="oppia-subtopic-name-text">{{ subtopic.key }}</label>
+                    <mat-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="let skill of searchInSubtopicSkills(subtopic.value, skillFilterText)" [value]="skill.id" aria-label="{{ skill.description }}">
+                      {{ skill.description }}
+                    </mat-radio-button>
+                  </mat-card>
+                </div>
+              </div>
+            </mat-card>
+          </div>
+          <mat-card *ngIf="untriagedSkillSummaries.length" class="list-view-item">
+            <mat-card layout="row" class="list-view-item">
+              <h2><label class="oppia-heading-text">Untriaged Skills</label></h2>
+            </mat-card>
+            <div *ngIf="!checkIfEmpty(untriagedSkillSummaries)" class="list-view-item ">
+              <mat-card class="list-view-item">
+                <mat-radio-button class="oppia-skills-list-item protractor-test-skills-list-item" *ngFor="let skill of searchInUntriagedSkillSummaries(skillFilterText)" [value]="skill.id" aria-label="{{ skill.description }}">
+                  <label>{{ skill.description }}</label>
+                </mat-radio-button>
+              </mat-card>
+            </div>
+          </mat-card>
+        </mat-radio-group>
+      </div>
+    </div>
+    <div class="list-group oppia-filter-list-group">
+      <mat-card *ngIf="allowSkillsFromOtherTopics" layout="row" class="list-view-item oppia-list-view-item">
```

Comment:
`layout="row"` here too.

Reply:
Done (FYI you can comment that it applies to all instances and save some time without going through the whole file)